### PR TITLE
Optionally run specified rocoto task as part of the ctests 

### PR DESCRIPTION
**Pull request by guillaumevernieres**
Only tested with the coupled UFS and the marine DA tasks:
```
Test project /scratch1/NCEPDEV/stmp2/Guillaume.Vernieres/runs/prs/global-workflow/sorc/gdas.cd/build/gdas/test/gw-ci
  Test  #1: C48mx500_3DVarAOWCDA
  Test  #2: C96_atmaerosnowDA
  Test  #3: C96C48_ufs_hybatmDA
  Test  #4: C48mx500_3DVarAOWCDA_gdasfcst_202103241200
  Test  #5: C48mx500_3DVarAOWCDA_gdasprepoceanobs_202103241800
  Test  #6: C48mx500_3DVarAOWCDA_gdasocnanalprep_202103241800
  Test  #7: C48mx500_3DVarAOWCDA_gdasocnanalbmat_202103241800
  Test  #8: C48mx500_3DVarAOWCDA_gdasocnanalrun_202103241800
  Test  #9: C48mx500_3DVarAOWCDA_gdasocnanalchkpt_202103241800
  Test #10: C48mx500_3DVarAOWCDA_gdasocnanalpost_202103241800

Total Tests: 10
```

### CODE REVIEW

**Review by AndrewEichmann-NOAA — DISMISSED** [review]
Looks like it could work

**Review by danholdaway — DISMISSED** [review]

File: test/gw-ci/CMakeLists.txt [hunk 1]
```diff
@@ -6,10 +6,34 @@ set(RUNTESTS ${CMAKE_CURRENT_BINARY_DIR}/../../test/gw-ci)
...
   set(YAML ${HOMEgfs}/ci/cases/pr/${pslot}.yaml)
   add_test(NAME ${pslot}
            COMMAND /bin/bash -c "${PROJECT_SOURCE_DIR}/test/gw-ci/create_exp.sh ${YAML} ${pslot} ${HOMEgfs} ${RUNTESTS}"
            WORKING_DIRECTORY ${RUNTESTS})
   set_tests_properties(${pslot} PROPERTIES LABELS "manual")
 endforeach()
+
+# Run tasks for C48mx500_3DVarAOWCDA
+# ----------------------------------
+set(pslot C48mx500_3DVarAOWCDA)
```
Comment (CoryMartin-NOAA) [inline test/gw-ci/CMakeLists.txt]: can we put this in the loop above? or a new loop

File: test/gw-ci/create_exp.sh [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
 
 # Get ICSDIR_ROOT
 source "${HOMEgfs}/ush/detect_machine.sh"
 source "${HOMEgfs}/ci/platforms/config.${MACHINE_ID}"
 
+# Arguments for the exp setup
+expyaml="$1"
+export pslot="$2"
+export RUNTESTS="$4"/${pslot}
+export SLURM_ACCOUNT="da-cpu"
```
Comment (CoryMartin-NOAA) [inline test/gw-ci/create_exp.sh:15]: so I'm guessing this is here to overwrite something in the CI config, but why are the $1 and $2 things moved down here?

**Review by CoryMartin-NOAA — APPROVED** [review]
Merci

**Pull request merged**